### ISSUE TITLE
Report feature failures better

### ISSUE THREAD

**Issue by cfjedimaster** [heremaps/here-cli#269]
I'm attempting to upload a large file of complex features (https://github.com/cfjedimaster/heredemos/blob/master/misc_related/countries.geojson) and the CLI reported:

```
uploading file - countries.geojson
uploaded feature count :55, failed feature count :200
55 features uploaded to Data Hub space 'oNpF9ndC' in 12.21 seconds, at the rate of 5 features per second
upload completed successfully
```

When I saw the last line, I figured all was well. I didn't notice the failed feature count at all. I'd suggest the CLI *greatly* amplify the fact that something went wrong, specifically in the last sentence as that's all a developer may check. Perhaps:

```
upload completed with failed features
```

**Comment by burritojustice** [comment]
You're right -- the "upload completed with failed features" needs too better reflect the state of the upload.

We've added some logic in the next version to automatically try a smaller chunk if it fails with a 413 (entity too large error) which will help minimize those errors. 

However, some of those features (specifically, Antarctica, Russia and Fiji) are failing because of coordinates greater or less than 180. This is common in geometries that cross the international dateline. This was common in the early days of geojson, but the [recommended solution is to convert the geometry to a multipolygon](https://macwright.com/2016/09/26/the-180th-meridian.html), split along the dateline, but this would require pre-processing in another geospatial tool.

Regardless, we'll improve the error messaging.

**Comment by burritojustice** [comment]
resolved in v1.6.0
